components/suggest/src/suggestion.rs

```diff
@@ -16,6 +16,11 @@ const TIMESTAMP_TEMPLATE: &str = "%YYYYMMDDHH%";
 /// 2 bytes shorter than [`TIMESTAMP_TEMPLATE`].
 const TIMESTAMP_LENGTH: usize = 10;
 
+/// Suggestion Types for Amp
+pub enum AmpSuggestionType {
```

Comment:
```suggestion
pub(crate) enum AmpSuggestionType {
```

Nit: Let's stick to `pub` for things that we expose in the UDL, and `pub(crate)` for things that we use within the crate.